docs/book/intro.md

```diff
@@ -53,6 +53,33 @@ that the session data won't be changed or when only one of the concurrent script
 may change it. The last script that changes and persists the session data will
 overwrite any previous change.
 
+### Enabling deletion of cookie in browser
+
+The default behaviour of the php session extension is to never delete the cookie
+in the browser. It is possible to automatically delete the cookie in the browser
+when the session becomes empty.
+
+This option can be enabled using the following configuration:
+
+```php
+// file: data/session.global.php
+return [
+    'session' => [
+        'persistence' => [
+            'ext' => [
+                'delete_cookie_on_empty_session' => true, // true|false
+            ],
+        ],
+    ],
+];
+```
+
+Then whenever you call `$session->clear();`, the cookie will be deleted from in
+the browser in addition to deleting session on the server.
+
+This is the recommended configuration to prevent leftover in the browser, unless
```

Comment:
I am not sure make recommendation to remove cookie on empty session is ok. It is always depends on what developer want to achieve.

Reply:
I removed the recommendation.